main.py

```diff
@@ -22,12 +24,32 @@ class Common(db.Model):
     color = db.Column(db.String(10))
     winrate = db.Column(db.Float)
     img_url = db.Column(db.String(1000))
+    timestamp = db.Column(db.DateTime)
 
 
-db.drop_all()
-if not inspect(db.engine).has_table("common"):
-    print("create table")
-    db.create_all()
+
+
+def Get_Common_In_DB():
```

Comment:
Funktionen schreibt man in Python klein
https://realpython.com/python-pep8/#naming-styles

Reply:
Hast du recht, weiß nicht warum ich die Anfangsbuchstaben großgeschrieben habe